contracts/script/LightClientTest.s.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "forge-std/Script.sol";
+import { BN254 } from "bn254/BN254.sol";
+import { LightClient as LC } from "../src/LightClient.sol";
+import { LightClientTest as LCTest } from "../test/mocks/LightClientTest.sol";
+
+contract DeployLightClientScript is Script {
```

Comment:
There now seem to be 3 scripts that do about the same thing. Would it be feasible to have one script and pass different inputs (via env vars, JSON file, ...) to achieve the same? If not I think there should be at least some comments explaining why we do it this way.

Reply:
agree, we should remove the duplication, maybe just leave the one for deploying the LightClientTest, for now?

Reply:
Agreed. I suggest we create a ticket as deployment becomes a bit more involved with upgradeable contracts.

Reply:
rn, we only have two scripts, which make sense, one is for deploying LightClientTest, the other is for demo and for actual production/staging.

Reply:
As discussed with  @mrain we will unify how the light contract is initialized/deployed for testing purposes. See #1092.